Bottom: 0057d86558560de79e853a7d30092764604c1a73
Top:    9cb95736016b778ed17f22425f45da2083121886
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-09 05:13:16 +0000

Refresh of attempt-to-fix-polygon-slownes

---

diff --git a/src/global.h b/src/global.h
index 1fb3420..157e186 100644
--- a/src/global.h
+++ b/src/global.h
@@ -261,12 +261,6 @@ typedef struct			/* holds information about arcs */
     Delta;
 } ArcType, *ArcTypePtr;
 
-typedef struct
-{
-  struct rtree_node *root;
-  int size;			/* number of entries in tree */
-} rtree_t;
-
 typedef struct			/* holds information about one layer */
 {
   char *Name;			/* layer name */
diff --git a/src/polygon1.c b/src/polygon1.c
index 336c54e..25155ab 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1980,8 +1980,6 @@ M_POLYAREA_update_primary (jmp_buf * e, POLYAREA ** pieces,
           if (curc->Flags.orient == PLF_DIR && !is_first)
             printf ("A:Hmm, got PLF_DIR orientation for a non-first contour\n");
 
-          if (curc->Flags.orient != PLF_DIR && is_first)
-            printf ("A:Hmm, got first contour without PLF_DIR orientation\n");
 //          printf ("Separating a hole (belonging to a moved contour)\n");
         } else {
           assert (0);
